### ISSUE TITLE
Problem extending syntax map for LESS files

### ISSUE THREAD

**Issue by floq-design** [SublimeCodeIntel/SublimeCodeIntel#145]
Hi,
I've tried adding LESS to the codeintel_syntax_map as found in the Base File.sublime-settings file as so:

<pre>
"codeintel_syntax_map":
{
    "Python Django": "Python",
    "LESS": "CSS"
}
</pre>

But I'm not getting code hints in less files.
Any ideas where I'm going wrong?
Cheers

**Comment by druellan** [comment]
Yeah. Same here. CSS/LESS is silently not working: no db geberation, no log messages of any kind.
I'm using CSS-Extended in the meantime, but I like the idea of centralize all into one plugin.
